resources/monitoring/templates/_helpers.tpl

```diff
@@ -25,68 +25,69 @@ The longest name that gets created adds and extra 37 characters, so truncation s
 {{- end -}}
 
 {{/* Fullname suffixed with operator */}}
-{{- define "prometheus-operator.operator.fullname" -}}
-{{- printf "%s-operator" (include "prometheus-operator.fullname" .) -}}
+{{- define "kube-prometheus-stack.operator.fullname" -}}
+{{- printf "%s-operator" (include "kube-prometheus-stack.fullname" .) -}}
 {{- end }}
 
 {{/* Fullname suffixed with prometheus */}}
-{{- define "prometheus-operator.prometheus.fullname" -}}
-{{- printf "%s-prometheus" (include "prometheus-operator.fullname" .) -}}
+{{- define "kube-prometheus-stack.prometheus.fullname" -}}
+{{- printf "%s-prometheus" (include "kube-prometheus-stack.fullname" .) -}}
 {{- end }}
 
 {{/* Fullname suffixed with alertmanager */}}
-{{- define "prometheus-operator.alertmanager.fullname" -}}
-{{- printf "%s-alertmanager" (include "prometheus-operator.fullname" .) -}}
+{{- define "kube-prometheus-stack.alertmanager.fullname" -}}
+{{- printf "%s-alertmanager" (include "kube-prometheus-stack.fullname" .) -}}
 {{- end }}
 
 {{/* Create chart name and version as used by the chart label. */}}
-{{- define "prometheus-operator.chartref" -}}
+{{- define "kube-prometheus-stack.chartref" -}}
 {{- replace "+" "_" .Chart.Version | printf "%s-%s" .Chart.Name -}}
 {{- end }}
 
 {{/* Generate basic labels */}}
-{{- define "prometheus-operator.labels" }}
-chart: {{ template "prometheus-operator.chartref" . }}
+{{- define "kube-prometheus-stack.labels" }}
+chart: {{ template "kube-prometheus-stack.chartref" . }}
 release: {{ $.Release.Name | quote }}
+heritage: {{ $.Release.Service | quote }}
```

Comment:
We dropped all the `heritage` labels as it is related to Helm 2, and we are using Helm 3. Please delete them.